aries-site/src/layouts/content/InPageNavigation.js

```diff
@@ -123,10 +144,20 @@ export const InPageNavigation = ({ headings }) => {
                         ? undefined
                         : { left: theme.global.borderSize.small }
                     }
+                    direction="row"
+                    align="top"
+                    gap="small"
                   >
                     <Text color="text-strong" size="small" weight="normal">
                       {headingTitle}
                     </Text>
+                    {newUpdate && (
+                      <Box justify="top">
```

Comment:
Is all of this nested Box and Text necessary? I would think we could just do:

```
{newUpdate && <StatusGoodSmall />}
```

Reply:
The spacing is a bit weird, but design-wise they wanted the status indicator to align at the center of the top line (in cases where it is multiple lines. So I put the icon in the <Text> component to keep the padding of the icon at line-height, and then I needed the <Box> component to register the justify=top to keep it align with the first line. I'm not 100% that this is the best way to do it, but that's what I was trying to get at with the nested components